sdk/containerinstance/Azure.ResourceManager.ContainerInstance/api/Azure.ResourceManager.ContainerInstance.netstandard2.0.cs

```diff
@@ -553,8 +553,8 @@ public partial class UsageName
     public partial class VolumeMount
```

Comment:
Is `VolumeMount` still colliding with other RPs?

Reply:
Updated